pages/common/gh-issue-create.md

```diff
@@ -0,0 +1,32 @@
+# gh issue
+
+> Create issues in GitHub from the command-line.
```

Comment:
```suggestion
> Create issues for GitHub from the command-line.
```

Reply:
Neither `in` nor `for` work well here.  `on` sounds the best, but I'd rearrange it so as to not have to use the preposition:
```suggestion
> Create GitHub issues on a repository from the command-line.
```